docs/earthfile/earthfile.md

```diff
@@ -751,6 +751,10 @@ This feature is currently in **Beta** and it has the following limitations:
 
 #### Options
 
+##### `-f <dockerfile-path>`
+
+Specify an alternative Dockerfile to use. The `<dockerfile-path>` can be either a path on the host system, relative to the current Earthfile, or an artifact reference pointing to the `Dockerfile`.
```

Comment:
```suggestion
Specify an alternative Dockerfile to use. The `<dockerfile-path>` can be either a path on the host system, relative to the current Earthfile, or an artifact reference pointing to a `Dockerfile`.
```

Reply:
So a Dockerfile as an artifact must be named +some-target/Dockerfile? At least, thats how I initially parsed this sentence. Assuming that i s not true, I would probably phrase it like this

Reply:
Side note: I went to add a link to what an "artifact reference" is. The closest I found is the description in `COPY`. This seems like a fundamental term I should find a better definition for when I search "artifact reference" over our docs.

Reply:
It doesn't have to be named `Dockerfile` - good catch.

Ah right.. it's in `target-ref.md`. I'll add a link to that too.